packages/webapp/src/server.js

```diff
@@ -47,6 +47,24 @@ const init = async server => {
   await server.register(Blipp)
   await server.register(onPreHandler)
   await server.register(onPostHandler)
+  await server.register({
+    plugin: header,
+    options: {
+      keys: [
+        { key: 'X-Frame-Options', value: 'deny' },
+        { key: 'X-Content-Type-Options', value: 'nosniff' },
+        { key: 'Access-Control-Allow-Origin', value: SERVICE_HOME_URL },
+        { key: 'Cross-Origin-Opener-Policy', value: 'same-origin' },
+        { key: 'Cross-Origin-Embedder-Policy', value: 'require-corp' },
+        { key: 'X-Robots-Tag', value: 'noindex, nofollow' },
+        { key: 'X-XSS-Protection', value: '1; mode=block' },
+        { key: 'Strict-Transport-Security', value: 'max-age=31536000;' },
```

Comment:
Please use the default max-age configured by Hapi.js - https://hapi.dev/api/?v=21.3.2#-routeoptionssecurity

Reply:
done